docker/compose_files/files/nginx.conf

```diff
@@ -99,19 +98,6 @@ http {
     #proxy_read_timeout 1200;
     #send_timeout 1200;
 
-    location /bundleservice {
```

Comment:
Why did you decide to merge /bundleservice with the root url / ? What did we have at / before?

Reply:
`/bundleservice` no longer exists - this is from the distant past.